configs/codec.go

```diff
@@ -5,13 +5,13 @@ import (
 
 	"github.com/BurntSushi/toml"
 
-	"github.com/projecteru2/yavirt/pkg/errors"
```

Comment:
actually, why not wraps the `cockroachdb/errors` within `yavirt/pkg/errors`, then don't need to change anything from upper pkgs.

Reply:
Using upstream package directly is enough and adding a thin wrapper is needless. actually I just want to unify the log and errors pkg of core,yavirt and some components developed by me.